.github/actions/publish-maester-module/action.yml

```
@@ -0,0 +1,51 @@
+name: Build and publish Maester module
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)